sonic_platform_base/module_base.py

```diff
@@ -262,28 +266,30 @@ def get_state_info(self):
         """
         raise NotImplementedError
 
-    def get_bus_info(self, module_name):
+    def get_bus_info(self):
         """
-        Retrieves the bus information for the specified by "module_name" on a SmartSwitch.
+        Retrieves the bus information.
 
         Returns:
             Returns the PCI bus information in BDF format like "[DDDD:]BB:SS:F"
         """
         raise NotImplementedError
 
-    def pci_detach(self, module_name):
+    def pci_detach(self):
         """
-        Detaches the DPU PCI device specified by "module_name" on a SmartSwitch.
+        Detaches the PCI device.
 
         Returns: True once the PCI is successfully detached.
+        Returns False, if PCI detachment fails or specified device is not found.
         """
         raise NotImplementedError
 
-    def pci_reattach(self, module_name):
+    def pci_reattach(self):
```

Comment:
@vvolam nit...rename to pci_rescan ?

Reply:
@prgeor,  Though I modified, wanted to rename back as pci_reattach is Linux kernel standard function name. To avoid confusion and conflicts better to avoid same names.